db/transaction/transaction.go

```diff
@@ -59,8 +59,8 @@ type ResourceState struct {
 	Monitoring    string
 }
 
-//ListOptions specifies additional list related options.
-type ListOptions struct {
+//ViewOptions specifies additional list related options.
```

Comment:
No.I used this tool.I think @p-kozlowski meant that "ViewOptions specifies additional list related options." didn't make sense anymore,that's why I changed this sentence by hand.